### ISSUE TITLE
[REQ]: Please add a timeout limit when checking bookmarks

### ISSUE THREAD

**Issue by spamtaztik** [fmd-project-team/FMD#907]
From time to time when my FMD was checking bookmarks, it would get stuck at one title and would stay there for, well, ever. I have to disable that particular entry for FMD to move on to the next bookmark. Strangely enough on the next round of bookmark check, that erroneous entry can be checked just fine. It used to be on titles from whitecloudpavilion, but I've never seen one like that for quite some time and I thought it was fixed. Today it happened again for a title in mangadex. 

TL;DR: can you add a timer when checking each bookmark entry (say 10, 15, 20 seconds?) and if the check doesn't produce any results just move on to the next one and put 0 to the "current chapter" column?

Thankees.

**Comment by dazedcat19** [comment]
Connection > timeout.
